packages/office-addin-lint/test/src/test.ts

```diff
@@ -11,12 +11,16 @@ describe("test cases", function() {
     it("eslint and prettier commands", async function() {
         const inputFile = "./test/cases/basic/functions.ts";
         const lintExpectedCommand = "./test/cases/basic/functions.ts";
+        const lintTestExpectedCommand = ".eslintrc.test.json"
```

Comment:
I found this name quite misleading. It is not really a Command, more like a file inside the command.
Would the change `lintTestExpectedCommand` to `lintTestExpectedCommandFile`.